src/libs/Launcher.js

```diff
@@ -1,5 +1,6 @@
+// @ts-check
 /* eslint-disable no-unused-vars */
-import { Q, models } from 'cozy-client'
+import CozyClient, { Q, models } from 'cozy-client'
```

Comment:
If `CozyClient` is imported for the JSDoc, you should remove it and use the `import('cozy-client').default` or something like that syntax.